lib/skylight/normalizers/elasticsearch/request.rb

```diff
@@ -0,0 +1,20 @@
+module Skylight
+  module Normalizers
+    module Elasticsearch
+      class Request < Normalizer
+        register "request.elasticsearch"
+
+        CAT = "db.elasticsearch.request".freeze
+
+        def normalize(trace, name, payload)
+          path = payload[:path].split('/')
+          title = [payload[:method], path[0]].compact.join(' ')
+          desc = {}
+          desc[:type] = path[1] if path[1]
+          desc[:id] = '?' if path[2]
```

Comment:
I guess it's just the id and the type, the type has a finite number of options, right?